Makefile.am

```diff
@@ -15,20 +14,16 @@ EXTRA_DIST = 			\
 	tomboy.spec.in		\
 	tomboy.spec			\
 	\
-	MAINTAINERS
+	tomboy.doap
```

Comment:
I would add this line in a separate commit, since changing from `MAINTAINERS` to `*.doap` is technically unrelated to porting away from `gnome-common`.